RELEASE-NOTES.md

```diff
@@ -15,6 +15,7 @@
 * `snow snowpark registry` was replaced with `snow registry` command.
 * Removed `snow streamlit create` command. Streamlit can be deployd using `snow streamlit deploy`
 * `snow connection test` now outputs all connection details (except for the password), along with connection status
+* Moved `snow stage` and `snow warehouse` from top-level to `snow object` subgroup
```

Comment:
We agreed to remove `snow warehouse status` (the only wh command) in favour of `snow object show warehouses`

Reply:
Ok, i was basing on table in https://docs.google.com/document/d/1FLkKWWsKLWVeJ4n0TWARi6voqinpeu54Tf85v1kY3yw/edit#heading=h.67voytek1ow7

Reply:
BTW. Should i add `snow object drop` etc. as part of this PR?

Reply:
Done